.github/workflows/docs.yml

```diff
@@ -8,30 +9,27 @@ on: [push, pull_request]
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ["ubuntu-latest"]
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.9]
+        python-version: ["3.10"]
 
-    name:
+    name: Documentation tests
     steps:
     - uses: actions/checkout@v3
```

Comment:
```suggestion
    - uses: actions/checkout@v4
```

Reply:
See #139 for this and other related updates.